core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -30,10 +30,11 @@ oppia.directive('audioTranslationBar', [
 
         $('.oppia-translation-tab').on('dragover', function(evt) {
           evt.preventDefault();
-          if (!scope.showDropArea) {
-            scope.showDropArea = true;
-            scope.$digest();
-          }
+          /* Don't show drag and drop area for user that is logged in
+            and cannot add translation. */
+          scope.showDropArea = !scope.userIsGuest &&
```

Comment:
```
scope.showDropArea = (
  !scope.userIsGuest && !scope.translationIsDisallowed);
```

Also change varname to scope.dropAreaIsAccessible (in this codebase, starting varnames with "show" implies they are functions rather than booleans)

Reply:
showDropArea to dropAreaIsAccessible ok got it.

Reply:
Not just the name change. Note the simplification plus parentheses as well in the suggested edit.

Reply:
done